app/concepts/todo_list/create.rb

```diff
@@ -0,0 +1,32 @@
+class TodoList::Create < Trailblazer::Operation
+  include Model
+  model TodoList, :create
+
+  contract do
+    property :user_id
+    property :name
+  end
+
+  def process(params)
+    validate(params[:todo_list]) do |todo_list|
+      @model = find_or_create_todolist(params)
```

Comment:
@model gets assigned by the Model mixin. We don't want to access it directly. You can use `model` without the `@` instead.